common/meta/app/src/schema/table.rs

```diff
@@ -247,6 +251,19 @@ impl Default for TableMeta {
     }
 }
 
+impl TableMeta {
+    pub fn set_cluster_keys_meta(mut self, cluster_key: String) -> Self {
+        self.cluster_keys.push(cluster_key);
+        self.default_cluster_key_id = Some(self.cluster_keys.len() as u32 - 1);
+        self
+    }
```

Comment:
This method is not a `set`, it is a `push`.
Maybe naming it with `push_cluster_key` would be better.